tox.ini

```diff
@@ -18,10 +20,15 @@ commands=python make_docs
 commands=tox -c {toxinidir}/ops/ -e unit
 
 [testenv:lint]
-commands=tox -c {toxinidir}/ops/ -e lint
+commands =
+  flake8 {toxinidir}
+  black --check {toxinidir}
 
 [testenv:format]
-commands=tox -c {toxinidir}/ops/ -e format
+commands =
+  black {toxinidir}
 
 [flake8]
-ignore=E402
+ignore=E402,W503
```

Comment:
where's the [`W503`](https://www.flake8rules.com/rules/W503.html)? -- not a show stopper just curious

Reply:
```
canonical/interface-tls-certificates/tls_certificates_common.py:97:13: W503 line break before binary operator
```

for

```python
    @property
    def _is_top_level_server_cert(self):
        return (
            self.cert_type == "server"
            and self.common_name == self._unit.received_raw["common_name"]
        )
```

Reply:
i guess it's a conflict that black and flake8 created?  +1

Reply:
yy, black likes the W503 way